File: Server/src/ProEventos.API/Models/Evento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.API.Models
{
    public class Evento
    {
        public int EventoId { get; set; }
        public string Local { get; set; }
        public DateTime DataEvento { get; set; } 
        public string Tema {get; set;}
        public int QtdPessoas { get; set; }
        public string Lote { get; set; }
        public string ImagemURL { get; set; }
    }
}

File: Client/ProEventos-App/src/app/models/Evento.ts
import { Lote } from "./Lote";
import { RedeSocial } from "./RedeSocial";
import { Palestrante } from "./Palestrante";

export interface Evento {
  id : number;
  local : string;
  dataEvento?: string;
  tema : string;
  qtdPessoas : number;
  imagemURL : string;
  telefone : number;
  email : string;
  lotes : Lote[];
  redesSociais : RedeSocial[];
  palestrantesEventos : Palestrante[];
}
